packages/snowpack/src/commands/paint.ts

```diff
@@ -132,12 +114,16 @@ export function paint(bus: EventEmitter, plugins: string[]) {
       process.stdout.write(colors.dim(`  Server starting…`) + '\n\n');
     }
     // Console Output
-    if (consoleOutput.length) {
+    let history = logger.getHistory();
+    if (history.length > MAX_CONSOLE_LENGTH) {
```

Comment:
We’re not truncating history in the logger itself; only when we write to `stdout`.

Reply:
can you do a quick check to make sure that this isn't too slow in use? At 1000+ messages, slicing and then destructuring and then logging on every paint will probably start to get noticeably slow (just a hunch). 

On a long enough time frame, we'll run out of memory (especially if you have a noisy plugin like svelte-check).